modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -258,9 +267,13 @@ protected int processStartTag() throws Exception {
 			}
 
 			if (Validator.isNotNull(_label)) {
-				String label = LanguageUtil.get(
-					TagResourceBundleUtil.getResourceBundle(pageContext),
-					_label);
+				String label = getLabel();
+
+				if (!_localizeTitle) {
+					label = LanguageUtil.get(
+						TagResourceBundleUtil.getResourceBundle(pageContext),
+						_label);
+				}
```

Comment:
This attribute determines if we are localizing `label`, not `title`. It should be called `localized`. Conversely, if we were localizing two different parts of the tag, we could call them something like `labelLocalized` and `footerLocalized`.